src/ostorlab/agent/agent.py

```diff
@@ -48,6 +47,10 @@ class MaximumDepthProcessReachedError(exceptions.OstorlabError):
     """The processing depth limit is enforced and reached the limit."""
 
 
+class AgentNotAcceptedError(exceptions.OstorlabError):
+    """The agent is not in the list of accepted agents."""
```

Comment:
Should not be an error. This will happen and is not an error. We just filter them out.